src/DashboardService.Mappers/Db/DbBoardMapper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.DashboardService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Board;
+using LT.DigitalOffice.Kernel.Extensions;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Mappers.Db;
+
+public class DbBoardMapper : IDbBoardMapper
+{
+  private readonly IHttpContextAccessor _httpContextAccessor;
+
+  public DbBoardMapper(IHttpContextAccessor httpContextAccessor)
+  {
+    _httpContextAccessor = httpContextAccessor;
+  }
+
+  public DbBoard Map(CreateBoardRequest request)
+  {
+    return new()
+    {
+      Id = Guid.NewGuid(),
+      ProjectId = request.ProjectId,
+      Name = request.Name,
+      IsActive = true,
+      CreatedBy = _httpContextAccessor.HttpContext.GetUserId(),
```

Comment:
@SolovyovDmitry Do we pass the user Id to the mapper or do it here?

Reply:
yep

Reply:
+